### ISSUE TITLE
Create Scripts are invalid (Create 0.3.1c)

### ISSUE THREAD

**Issue by CyanSaber** [Vazkii/crucial2#131]
![2021-05-02_10 52 11](https://user-images.githubusercontent.com/83536637/116809976-03ca3080-ab39-11eb-8f61-57ae40e7d6d8.png)

**Comment by CyanSaber** [comment]
This is a fresh install with Create 0.3.1c

**Comment by Vazkii** [comment]
I've been told CreateTweaker also needs to be installed. Next version will warn you about it if it's not included.

**Comment by CyanSaber** [comment]
Can confirm, no more errors now, thanks 👍

**Comment by Vazkii** [comment]
Keeping it open as a caution until it's fixed.

**Comment by tyreforhyred** [comment]
i know that the changelog said the scripts for this were fixed since 1.2.5b but i still haven't been able to get it working.
this is on Crucial 2 version 1.3 with:

create-mc1.16.5_v0.3.2d
CreateTweaker-1.0.0.18
ContentTweaker-1.16-1.0.0.develop.19
as the only additional mods added in manually, is there something i'm missing here?

![2021-08-11_22 31 42](https://user-images.githubusercontent.com/76914208/129129984-74e54cae-be6e-4691-b685-5c60960bdbce.png)

**Comment by Beetrootmonkey** [comment]
@tyreforhyred I'm experiencing the same exact issue. Curse Forge seems to prefer Create v0.3.2c for some reason, but since the script is the same I'm getting the same error. I got it to work eventually by replacing every instance of <fluid:create:milk> with <fluid:minecraft:milk> (note the "minecraft" instead of the "create", since the fluid seems to be registered differently than expected by the script).

**Comment by Vazkii** [comment]
Yeah, we came to this conclusion over on discord too. The scripts were made
for 0.3.2 so they don't work with the latest fluid changes. I'll take care
of it either later today or on Friday.
